tests/kubeUtils.sh

```diff
@@ -62,10 +62,19 @@ function deploy_bookinfo() {
 }
 
 function find_ingress_controller() {
-    local ip="$(${K8CLI} get svc istio-ingress-controller -n ${NAMESPACE} \
-      -o jsonpath='{.status.loadBalancer.ingress[*].ip}')"
-    echo ${ip[@]}
-    [[ ${ip} =~ ^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$ ]] && return 0
+    #local gateway="$(${K8CLI} get svc istio-ingress-controller -n ${NAMESPACE} \
```

Comment:
I wasn't sure if we wanted to make it work either way (driven by some option) so I left the loadbalancer version there for now. If we don't need it for sure, we can take out in a future PR.